aidial_adapter_openai/app.py

```diff
@@ -125,38 +117,53 @@ async def chat_completion(deployment_id: str, request: Request):
     # Azure and non-Azure deployments.
     # Therefore, we provide the "model" field for all deployments here.
     # The same goes for /embeddings endpoint.
-    data["model"] = deployment_id
+    data["model"] = (
```

Comment:
Need to add possibility to pass model manually, since we have test case like this: `Test/Test-Model-QA`

Reply:
`data.get("model") or deployment_id`

Reply:
Fixed